modules/multilanguage/src/main/scala/com/ukg/surge/multilanguage/MultilanguageSidecarMain.scala

```diff
@@ -4,11 +4,38 @@ package com.ukg.surge.multilanguage
 
 import akka.actor.ActorSystem
 import akka.event.Logging
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+
+import scala.concurrent.ExecutionContext
+import scala.util.{ Failure, Success }
 
 class MultilanguageSidecarMain
 
 object MultilanguageSidecarMain extends App {
+
   implicit val system = ActorSystem("multilanguage")
+  implicit val ec: ExecutionContext = system.dispatcher
+
   val logger = Logging(system, classOf[MultilanguageSidecarMain])
-  val binding = new MultilanguageGatewayServer(system).run()
+  val multilanguageServer = new MultilanguageGatewayServer(system)
+
+  val route = path("healthz") {
+    get {
+      onComplete(multilanguageServer.doHealthCheck()) {
+        case Success(_)  => complete(StatusCodes.OK)
```

Comment:
With the response here I think we can look at whether or not the status is up or down to determine status code we reply with in addition to the success/failure of the future

Reply:
Actually, I did this because of kubernetes livenessProbe. 
It will just check the status code of the response. 
And I've updated the doHealthCheck.